dbt/models/staging/stg_costs.sql

```diff
@@ -0,0 +1,27 @@
+WITH raw_costs AS (
+
+    SELECT *
+    FROM raw.loading.raw_costs
```

Comment:
Did you check out [sources](https://docs.getdbt.com/docs/building-a-dbt-project/using-sources/) at all? If it's simple to implement, it might be worth making these tables into sources!

This is a pretty new feature so Im not super familiar!

Reply:
I'm adding them in!